src/main/java/org/wildfly/swarm/plugin/process/FractionManifestGenerator.java

```diff
@@ -66,6 +70,10 @@ public FractionMetadata apply(FractionMetadata meta) {
                         .map(DependencyMetadata::toString)
                         .collect(Collectors.toList())
             );
+            put("maven-dependencies",
```

Comment:
Could you add a comment that the previous `dependencies` and `transitive-dependencies` are "module dependencies per the JBos Modules `module.xml` declarations" or something like that? Just for people who might be coming to the code in the future.

Reply:
done